sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AuthTokenContext.java

```diff
@@ -46,6 +46,12 @@
     @VisibleForTesting
     static final int TOKEN_HISTORY_LIMIT = 5;
 
+    /**
+     * The minimum length of account Id.
```

Comment:
This is maximum length because when we use it on line 172 we're using it as a parameter for min, which means we take the shorter of the two substring lengths.

```suggestion
     * The Maximum length of account Id.
```